critiquebrainz/frontend/views/moderators.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+from critiquebrainz.data.model.user import User
+from custom_config import ADMINS
+
+moderators_bp = Blueprint('moderators', __name__)
+
+
+@moderators_bp.route('/moderators')
```

Comment:
Perhaps this should be just `/` instead since you already add a prefix in the `create_app` function. Otherwise path to this view is `/moderators/moderators`, which is unnecessary.